components/combobox/apiExamples/externalSelection.js

```diff
@@ -0,0 +1,19 @@
+import { AuroCombobox } from "../src/auro-combobox";
+
+/**
+ * Sets up demo  external selection in the combobox.
```

Comment:
Wording seems off, should be something like "Demonstrates how to make external selection in the combobox."